tests/player.test.js

```diff
@@ -110,4 +127,37 @@ describe('Player:', function () {
     assert.equal(browser.getText('#button-like'), '')
     assert.equal(browser.getText('#button-dislike'), '1')
   })
+
+  it('should play/pause a video when the spacebar is pressed', () => {
+    browser.url('http://localhost:3000/play/12345?playlist=98765')
+    browser.waitForClickable('#play-pause-button')
+    browser.switchTab()
```

Comment:
This, at least locally, gives the browser focus (if it was previously in the background, which it typically is) and as a result allows keypresses to successfully register via the `browser.keys` api.